src/pudl/extract/excel.py

```diff
@@ -167,24 +168,25 @@ def process_final_page(df, page):
         return df
 
     @staticmethod
-    def get_dtypes(year, page):
-        """Provide custom dtypes for given page and year."""
+    def get_dtypes(partition, page):
+        """Provide custom dtypes for given page and partition."""
         return {}
 
-    def extract(self, years):
+    def extract(self, **partitions):
         """Extracts dataframes.
 
         Returns dict where keys are page names and values are
         DataFrames containing data across given years.
 
         Args:
-            years (list): list of years to extract.
+            partitions (list, tuple or string): list of partitions to
```

Comment:
I tried using `**partitions` format here for the docs but i got a flake8 error. Is the right way to document kwargs to just list them like normal?... that seems weird.